blocks/api/serializer.js

```diff
@@ -102,7 +102,7 @@ export function getCommentAttributes( allAttributes, blockType ) {
 		}
 
 		// Ignore values sources from content and post meta
-		if ( attributeSchema.source || attributeSchema.meta ) {
+		if ( attributeSchema.source !== 'comment' ) {
```

Comment:
Will this change cause meta attributes to be serialized?

Reply:
no this function should only take care of `comment` attributes, so better be explicit about it.